akka-actor/src/main/scala/akka/pattern/BackoffSupervisor.scala

```diff
@@ -23,15 +22,15 @@ object BackoffSupervisor {
    * most exceptions will immediately restart the child. You can define another
    * supervision strategy by using [[#propsWithSupervisorStrategy]].
    *
-   * @param childProps the [[akka.actor.Props]] of the child actor that
```

Comment:
since `BackoffSupervisor` is now deprecated it would be good to change the doc links `[[BackoffSupervisor]]` to plain `BackoffSupervisor` (with backticks)

several places in this file